### PR TITLE
Changing readme for the latest changes

### LINKED ISSUE #250: Create new readme to reflect changes in authn/authz
Whenever I finish authentication and authorization, I will come back to this issue

### PR DESCRIPTION
**Pull request by haroonmoh**
Fixes #250

### CODE REVIEW

File: control-plane/README.md [hunk 1]
```diff
@@ -16,6 +16,19 @@ The job control plane is used for controlling and monitoring aggregation jobs.
 * Run `firebase use --add` to add an alias. When prompted, select your **Project ID**, then give your Firebase project an alias. For more info, visit this [link](https://firebase.google.com/docs/cli#add_alias).
 * Run `yarn start` or `yarn deploy`
 
+## Making an Admin Account
+* After getting set up and deploying the JCP, go to the sign up page and create an account with the email "admin@chromium.org" and a password of your choice.
```
Comment (taoliaoleo) [inline control-plane/README.md:20]: We should allow the users to specify this account when they set up the control plane. I know this is hard coded for now, but could you create an issue for it?

File: control-plane/README.md [hunk 2]
```diff
@@ -41,4 +54,4 @@ Created and Updated sorts are much simpler and only require a single query to th
 A user can search for jobs by Job ID. To search for multiple jobs, separate with semicolons.
 
 ## Table
-The table UI is based on [Material Design Lite (MDL)](https://getmdl.io/). The top level of the table contains information about the job. When you click on the job, it will display information about the sub-jobs. 
+The table UI is based on [Material Design Lite (MDL)](https://getmdl.io/). The top level of the table contains information about the job. When you click on the job, it will display information about the sub-jobs.
```
Comment (taoliaoleo) [inline control-plane/README.md:57]: MDL probably uses Apache-2 license. Could you verify that?

File: control-plane/README.md [hunk 3]
```diff
@@ -16,6 +16,19 @@ The job control plane is used for controlling and monitoring aggregation jobs.
 * Run `firebase use --add` to add an alias. When prompted, select your **Project ID**, then give your Firebase project an alias. For more info, visit this [link](https://firebase.google.com/docs/cli#add_alias).
 * Run `yarn start` or `yarn deploy`
 
+## Making an Admin Account
```
Comment (taoliaoleo) [inline control-plane/README.md:19]: We should explain the roles and how the auth/authz works, i.e. users can only browse the jobs after the admin grants permissions.

**Review by taoliaoleo — CHANGES_REQUESTED** [review]

File: control-plane/README.md (hunk 2 above)
Comment (haroonmoh) [reply control-plane/README.md:57]: It does use Apache-2

File: control-plane/README.md (hunk 1 above)
Comment (haroonmoh) [reply control-plane/README.md:20]: I created an issue

File: control-plane/README.md [hunk 4]
```diff
@@ -27,7 +40,7 @@ The job control plane is used for controlling and monitoring aggregation jobs.
 * `yarn deploy-dev` - deploys dev code to firebase hosting
   * Requires Firebase to be setup in project
 
-# How it works
+# How it Works
```
[the remark this answers is not in the document]
Comment (haroonmoh) [reply control-plane/README.md:43]: Done

**Comment by haroonmoh** [comment]
Not fully done with the Authentication / Authorization part but I committed what I have so far

File: control-plane/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hostirosti) [reply control-plane/README.md]: Previous version was ok. For headers just the first word and Brand names (e.g. Google Cloud, Firebase) are capitalized.

File: control-plane/README.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 
-# Job Control plane
+# Job Control Plane
 
-The job control plane is used for controlling and monitoring aggregation jobs.
+The job control plane (JCP) is used for controlling and monitoring aggregation jobs.
 
-## Getting set up
+## Getting Set Up
```
Comment (hostirosti) [inline control-plane/README.md:6]: Prerequisites

File: control-plane/README.md (hunk 3 above)
Comment (hostirosti) [inline control-plane/README.md:19]: Set up admin account

File: control-plane/README.md [hunk 6]
```diff
@@ -16,6 +16,19 @@ The job control plane is used for controlling and monitoring aggregation jobs.
 * Run `firebase use --add` to add an alias. When prompted, select your **Project ID**, then give your Firebase project an alias. For more info, visit this [link](https://firebase.google.com/docs/cli#add_alias).
 * Run `yarn start` or `yarn deploy`
 
+## Making an Admin Account
+* After getting set up and deploying the JCP, go to the sign up page and create an account with the email "admin@chromium.org" and a password of your choice.
+* This should create an admin account. Refresh the page if you are not directed to the jobs table
+* To add or remove users, navigate to your user icon at the top right of the screen and a dropdown should appear. Click the users tab.
+
+## Allowing Google Sign In
```
Comment (hostirosti) [inline control-plane/README.md:24]: Allowing Google Auth login

File: control-plane/README.md [hunk 7]
```diff
@@ -16,6 +16,19 @@ The job control plane is used for controlling and monitoring aggregation jobs.
...
+* Navigate to the **Auth** tab, in the [Firebase Console](https://console.firebase.google.com/)
+* **Enable** Google sign in and click **Save**
+
+## Allowing GitHub Sign In
```
Comment (hostirosti) [inline control-plane/README.md:28]: Allowing Github Auth login

File: control-plane/README.md [hunk 8]
```diff
@@ -29,6 +42,16 @@ The job control plane is used for controlling and monitoring aggregation jobs.
...
+Authentication is provided by [Firebase Authentication](https://firebase.google.com/docs/auth) while Authorization lies inside of [Cloud Firestore](https://firebase.google.com/docs/firestore) and [Firestore Rules](https://firebase.google.com/docs/firestore/security/get-started). We have four roles for all authenticated users:
+
+* **Admin** - able to view/edit jobs and manage users
+* **Editor** - able to view/edit jobs
+* **Viewer** - able to view jobs
+* **Pending** - able to sign in but not able to view/edit jobs
+
+All authenticated users start as a **Pending** user and must be assigned a role by an **Admin** user.
+
 ## Main Architecture
```
Comment (hostirosti) [inline control-plane/README.md:55]: Main architecture

File: control-plane/README.md [hunk 9]
```diff
@@ -41,4 +64,4 @@ Created and Updated sorts are much simpler and only require a single query to th
 A user can search for jobs by Job ID. To search for multiple jobs, separate with semicolons.
 
 ## Table
```
Comment (hostirosti) [inline control-plane/README.md:66]: User table

**Review by hostirosti — CHANGES_REQUESTED** [review]

**Review by taoliaoleo — APPROVED** [review]

**Review by hostirosti — APPROVED** [review]

**Pull request merged**